CONTRIBUTING.md

```diff
@@ -22,38 +22,58 @@ To run the test suite, call `./test.sh` from your local machine - the script wil
 
 ## Testing
 
-### Running Tests
+### Running Unit Tests
 
-To run the service broker unit tests run:
+To run the Conjur Service Broker unit tests, run the following:
 ```sh-session
 $ # Build the base images and artifacts
 $ ./build.sh
 
-$ # Run the tests
-$ ./test.sh
+$ # Run the unit tests
+$ ./bin/test_unit
 ```
 
-_Note: Tests rely on having built `conjur-service-broker` and `conjur-service-broker-test`
+### Running Local Integration Tests
+
+The [bin/test_integration](./bin/test_integration) script provides a full suite of integration tests
+for testing Service Broker functionality against Conjur. The tests use Docker
+Compose to spin up local instances of Conjur and Service Brokers, so the
+tests can be run locally.
+
+To run the Service Broker local integration tests, run the following:
+```sh-session
+$ # Build the base images and artifacts
+$ ./build.sh
+
+$ # Run the integration tests
+$ ./bin/test_integration
+```
```

Comment:
Suggestion: 

To run the Conjur Service Broker local integration tests, first build the base image and artifacts:
```shell script
./build.sh
```
Then, run the tests with the following command:
```shell script
./bin/test_integration
```